source/message/OP_MSG.rst

```diff
@@ -206,6 +215,38 @@ connection. All other errors during processing will be silently dropped, and
 will not result in the connection being closed.
 
 
+moreToCome On Responses
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When the ``moreToCome`` flag is set on a response it signals to the recipient
+that the sender will send additional responses on the connection. The recipient
+MUST continue to read responses until it reads a response with the ``moreToCome``
+flag not set, and MUST NOT send any more requests on this connection until
+it reads a response with the ``moreToCome`` flag not set. The client MUST
+either consume all messages with the ``moreToCome`` flag set or close the connection.
+
+When the server sends responses with the ``moreToCome`` flag set,
+each of these responses will have a unique ``messageId``, and the
+``responseTo`` field of every follow-up response will be the ``messageId`` of
+the previous response.
+
+The client MUST be prepared to receive a response without ``moreToCome`` set
+prior to completing iteration of a cursor, even if an earlier response for
+the same cursor had ``moreToCome`` flag set. To continue iterating such a cursor,
+the client MUST issue an explicit ``getMore`` request.
+
+
+exhaustAllowed
```

Comment:
The server opmsg spec that I read does not restrict what exhaustallowed may be set on. This PR is for the wire protocol, and the wire protocol only says that if exhaustallowed is set the other side may use moretocome. That's it.

When it makes sense to set exhaustallowed is a separate question, and one that is tackled in spec-1155.

Reply:
If I moved the sections their order in the table won't match their order in the descriptions. If you really want the sections in the other order I'll make this change.

Reply:
> The server opmsg spec that I read does not restrict what exhaustallowed may be set on. This PR is for the wire protocol, and the wire protocol only says that if exhaustallowed is set the other side may use moretocome. That's it.

If you think this section should be a copy of https://docs.mongodb.com/manual/reference/mongodb-wire-protocol/#op-msg, I guess that's ok.  If so, I still think it would be useful to talk about cursors/getMore somewhere; perhaps as a separate section, similar to https://github.com/mongodb/specifications/blob/master/source/message/OP_MSG.rst#command-arguments-as-payload.

Happy to talk it over in person if you'd like.

Reply:
@jyemin Added server support.